### ISSUE TITLE
Feature Request: Provide a compatible custom exporter for TexturePacker

### ISSUE THREAD

**Issue by rparrett** [ickshonpe/bevy_heterogeneous_texture_atlas_loader#2]
It seems like this should be doable: https://www.codeandweb.com/texturepacker/documentation/custom-exporter#texturepackers-custom-exporter-format

**Comment by ickshonpe** [comment]
Yep should be straightforward. I wrote the crate because I saw someone asking on Discord about importing irregular texture atlases and I used the format from their custom engine. It'd be worth doing to get Bevy on their list of supported game engines.

I'll have a go when I'm updating the crate for version 0.9

**Comment by ickshonpe** [comment]
Yep should be straightforward. It'd be good to get Bevy on their list of supported game engines.

I'll have a go when I'm updating the crate for Bevy 0.9
